immaculateGridCalculations/gridSolver.py

```diff
@@ -255,7 +265,8 @@ def getPlayerSeasonK(minK):
     query = (
         db.session.query(
             Pitching.playerID,
-            Pitching.teamID.label("teamID")
+            Pitching.teamID.label("teamID"),
+            Batting.yearID.label("yearID")
         )
         .group_by(Pitching.playerID, Pitching.yearID, Pitching.teamID)  # Group by playerID, yearId, and teamID
```

Comment:
Wouldn't Pitching.yearID need to be replaced by Batting.yearID in the group_by statement?

Reply:
Ahh, good catch, I missed that while copy pasting!